[2023-11-02T18:18:07.093+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-11-02T18:18:02.564278+00:00 [queued]>
[2023-11-02T18:18:07.109+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-11-02T18:18:02.564278+00:00 [queued]>
[2023-11-02T18:18:07.110+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-02T18:18:07.111+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-11-02T18:18:07.111+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-02T18:18:07.131+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-11-02 18:18:02.564278+00:00
[2023-11-02T18:18:07.140+0000] {standard_task_runner.py:55} INFO - Started process 102 to run task
[2023-11-02T18:18:07.143+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-11-02T18:18:02.564278+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp9y188h7g']
[2023-11-02T18:18:07.145+0000] {standard_task_runner.py:83} INFO - Job 29: Subtask is_api_available
[2023-11-02T18:18:07.214+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-11-02T18:18:02.564278+00:00 [running]> on host 6c8be2bfe956
[2023-11-02T18:18:07.303+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-11-02T18:18:02.564278+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-02T18:18:02.564278+00:00
[2023-11-02T18:18:07.305+0000] {http.py:120} INFO - Poking: api/
[2023-11-02T18:18:07.317+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-11-02T18:18:07.319+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-11-02T18:18:07.572+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-11-02T18:18:07.587+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20231102T181802, start_date=20231102T181807, end_date=20231102T181807
[2023-11-02T18:18:07.637+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-11-02T18:18:07.684+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
